File: js/singles/iteration.js
// init some variables
let y = 0;
let i = 0; 
let z = 0;

// run 10 times
for (i = 0; i < 10; i++) {
    y = y + 1; 
}
// y is now 10

// 10 times as y is 10
while (y > 0) {
    y = y - 1; // decrement by 1
    z = z + 2; // increment by 2
}

// z is now 20
console.log(z);

File: py/singles/iteration.py
# init y and z to zero
y = 0
z = 0
# 10 times
for i in range(10): 
    y = y + 1 # increment y by one
# here is y = 10 

# and bring y to zero again
# y is 10, so this runs 10 times
while y > 0:
    y = y - 1   # decrement y by one
    z = z + 2   # increment z by two

# thus, z is 20
print(z)
